.github/workflows/tests.yaml

```diff
@@ -127,19 +132,25 @@ jobs:
         run: |
           google-chrome --version
           xvfb-run --server-args="-screen 0, 1280x720x24" --auto-servernum \
-          chromedriver --port=4444 --url-base=/wd/hub &> ./logs/chromedriver.log &
+          chromedriver --port=4444 &> ./logs/chromedriver.log &
 
       - name: Start GeckoDriver
         if: ${{ !matrix.selenium-server && matrix.browser == 'firefox' }}
         run: |
           firefox --version
           geckodriver --version
           xvfb-run --server-args="-screen 0, 1280x720x24" --auto-servernum \
-          geckodriver &> ./logs/geckodriver.log &
+          geckodriver --port=4444 &> ./logs/geckodriver.log &
```

Comment:
Why this change? The port `4444` is default

Reply:
reverted